app/controllers/Contributions.scala

```diff
@@ -57,6 +57,7 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
       val intCmp = request.getQueryString("INTCMP")
       val refererPageviewId = request.getQueryString("REFPVID")
       val refererUrl = request.headers.get("referer")
+      val platform = request.getQueryString("platform")
```

Comment:
Ideally I'd like to see a platform agnostic flag to be passed to the template.
Something like `showHeader` would be a good idea.

Also, we have a utility function: `request.isAndroid` would do the trick